### ISSUE TITLE
[IDEA] Nested methods.

### ISSUE THREAD

**Issue by ghost** [AnErrupTion/LoGiC.NET#6]
Ofc this won't work because nested methods bad but here's an idea I've always had for c# obfuscation:

```cs
void Main(string[] args) {
    Console.WriteLine("Hello, World!");
}
Main(args);
```

Now you may think "oh that's not that bad", but link multiple nested methods together and combine them with other existing obfuscation techniques and everything is chaos. Another idea (ofc its probably implemented) is to abuse interpolated strings, eg:

```cs
string iliilllli = $"{$"{$"{$"Hello, World!"}"}"}";
```

Now you see how insane this can get? Once again combine this with the current obfuscation to get an insane mess.

**Comment by AnErrupTion** [comment]
Yeah you can make some sort of proxy methods, basically nested methods as you said. For the second idea, well I didn't think of that, I'll definitely try to do that!
